R/timeseriessummary-data.R

```diff
@@ -524,6 +525,151 @@ parseTSSGapTolerances <- function(reportData, timezone){
   return(gapTolerances)
 }
 
+#' Unnest the correction parameters
+#' @description Takes the corrections and unnests them so they can 
+#' more easily be processed for each type
+#' @param corrections the corrections for the report
+#' @param timezone the timezone to parse data into
+#' @return corrections formatted better for report display
+#' @importFrom dplyr rowwise mutate ungroup
+unNestCorrectionParameters <- function(corrections, timezone) {
+  
+  type <- ".dplyr"
+  driftPoints <- ".dplyr"
+  value <- ".dplyr"
+  startShiftPoints <- ".dplyr"
+  endShiftPoints <- ".dplyr"
+  usgsType <- ".dplyr"
+  upperThresholdPoints <- ".dplyr"
+  resamplePeriod <- ".dplyr"
+  gapLimit <- ".dplyr"
+  deviationValue <- ".dplyr"
+  deviationType <- ".dplyr"
+  windowSizeInMinutes <- ".dplyr"
+  ungroup <- ".dplyr"
+  
+  params <- corrections$parameters
+  corrections$parameters <- NULL
+  
+  if (length(params)>0) {
```

Comment:
this was where I made a comment before that might have been lost in the shuffle - could use `isEmptyVar` here. Up to you! I don't remember if we were trying to use those functions everywhere or not.